.github/actions/cl2-modules/cilium-metrics.yaml

```diff
@@ -91,3 +94,54 @@ steps:
         - name: Perc50
           query: quantile(0.5, cilium_agent_bootstrap_seconds_sum{scope="overall"})
           threshold: {{$MEDIAN_BOOTSTRAP_THRESHOLD}}
+
+    - Identifier: WatchRequestThresholds
+      Method: GenericPrometheusQuery
+      Params:
+        action: {{$action}}
+        metricName: Watch request thresholds
+        metricVersion: v1
+        unit: count
+        enableViolations: true
+        queries:
+        - name: Services
+          query: sum(apiserver_longrunning_requests{scope="cluster", verb="WATCH", resource="services"})
```

Comment:
Yeah, makes sense. The only risk would be in case of relists, but they should never happen in this context, and if they happen is probably a symptom of other problems like blocked informers.

Reply:
I will resolve this comment for now, it's better to start with some coverage and then improve it.
I also want to get confidence in CI if watch thresholds make sense and don't flake.